staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1beta1/types.go

```diff
@@ -33,8 +33,7 @@ type EgressSelectorConfiguration struct {
 // EgressSelection provides the configuration for a single egress selection client.
 type EgressSelection struct {
 	// name is the name of the egress selection.
-	// Currently supported values are "controlplane", "master", "etcd" and "cluster"
-	// The "master" egress selector is deprecated in favor of "controlplane"
```

Comment:
I would expect this to be supported until the v1beta1 type is removed. It can be converted to `controlplane` in conversion so all the apiserver code can only handle `controlplane`

Reply:
make sense.